src/it/java/com/google/checkstyle/test/base/AbstractGoogleModuleTestSupport.java

```diff
@@ -110,4 +135,77 @@ protected static List<Configuration> getModuleConfigsByIds(String... moduleIds)
         return getModuleConfigsByIds(CONFIGURATION, moduleIds);
     }
 
+    /**
+     * Performs verification of the file with the given file path against the whole google config.
+     *
+     * @param filePath file path to verify.
+     * @throws Exception if exception occurs during verification process.
+     */
+    public void verifyWithWholeGoogleConfig(String filePath) throws Exception {
+        final Configuration[] children = CONFIGURATION.getChildren();
+        final DefaultConfiguration root = new DefaultConfiguration(ROOT_MODULE_NAME);
+
+        for (Configuration child : children) {
+            root.addChild(child);
+        }
+
+        verifyWithItConfig(root, filePath);
+    }
+
+    /**
+     * Performs verification of the file with the given file path against google config.
+     * It uses the specified list of modules to load them from google config for validation.
+     *
+     * @param listOfModules list of modules to load from google config.
+     * @param filePath file path to verify.
+     * @throws Exception if exception occurs during verification process.
+     */
+    public final void verifyWithGoogleConfigParser(String[] listOfModules,
+           String filePath) throws Exception {
+        final DefaultConfiguration googleConfig = new DefaultConfiguration(ROOT_MODULE_NAME);
+        final DefaultConfiguration treeWalkerConfig =
+                new DefaultConfiguration(TreeWalker.class.getSimpleName());
+
+        for (String module : listOfModules) {
+            final List<Configuration> children =
+                    getConfigChildren(module);
+            if (CHECKER_CHILDREN.contains(module)) {
+                for (Configuration child : children) {
+                    googleConfig.addChild(child);
+                }
+            }
+            else {
+                for (Configuration child : children) {
+                    treeWalkerConfig.addChild(child);
+                }
+            }
+        }
+
+        if (treeWalkerConfig.getChildren().length > 0) {
+            googleConfig.addChild(treeWalkerConfig);
+        }
+
+        verifyWithItConfig(googleConfig, filePath);
```

Comment:
Same as other `verifyWithItConfig` comment.

Reply:
This is also dropped for the same reason, but also this whole method should be removed eventually when we switch everything to the whole config.

Reply:
@Zopsss Where is issue to eventually go back to convert other tests? This should be added to the end there and a comment `until http://....` should be added to this method.

Reply:
https://github.com/checkstyle/checkstyle/issues/14937

Reply:
added `until` message

https://github.com/checkstyle/checkstyle/blob/1b71a042246823ec7fdc3a752575c900c2b2aa14/src/it/java/org/checkstyle/base/AbstractItModuleTestSupport.java#L459-L467

Reply:
@Zopsss `until` was placed in wrong location. `until` is basically reminding us the mentioned method is to be removed. `verifyWithItConfig` is the way we will be proceeding now (unless something comes up), so this method should stay.

The note should be on `verifyWithConfigParser` in `AbstractGoogle`. We are only keeping it until all tests move over to full config testing. Once we do, it can be removed completely.

Reply:
done